fronts-client/integration/server/index.html

```diff
@@ -63,8 +63,11 @@
         lastName: 'Beddington',
         sentryPublicDSN:
           'https://4527e03d554a4962ae99a7481e9278ff@app.getsentry.com/35467',
-        mediaBaseUrl: 'https://media.gutools.co.uk',
-        apiBaseUrl: 'https://api.media.test.dev-gutools.co.uk',
+        baseUrls: {
+          mediaBaseUrl: 'https://media.gutools.co.uk',
```

Comment:
Given the other two are test urls, should this be `https://media.test.dev-gutools.co.uk` ?